src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    private static final Map<String, Integer> fruits = new HashMap<>();
+
+    public static Map<String, Integer> getFruits() {
+        return fruits;
+    }

```

Comment:
this defeats the purpose of having a FruitDao, if you can call the map directly. Try to remove this getter, and for all the places that used it, add a required functionality to Dao